File: Java/SendMessage/app/src/main/java/become/sk/opendetail/DisplayMessageActivity.java
package become.sk.opendetail;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.TextView;

public class DisplayMessageActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        // 1
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_display_message);

        // 2
        setTitle("Správa");

        // 3
        Intent intent = getIntent();
        String message = intent.getStringExtra(MainActivity.KEY);

        // 4
        TextView textView = (TextView) findViewById(R.id.textView);
        textView.setText(message);
    }
}

File: Kotlin/SendMessage/app/src/main/java/become/sk/opendetail/DisplayMessageActivity.kt
package become.sk.opendetail

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView

class DisplayMessageActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        // 1
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_display_message)

        // 2
        title = "Správa"

        // 3
        val intent = intent
        val message = intent.getStringExtra(MainActivity.KEY)

        // 4
        val textView = findViewById(R.id.textView) as TextView
        textView.text = message
    }
}
